docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,724 @@
+
+### submitTransaction
+
+This function runs repeatedly in the benchmark test phase. We will be evaluating the `queryCar` smart contract function by querying the assets we created in the `initializeWorkloadModule` method. 
+
+First, create a string identity for the asset to query, formed by the concatenation of the worker index and a random integer between 0 and the number of created assets. 
+
+Then return the call on `querySmartContract`, passing context, contract name, contract version, and an object that contains: `chaincodeFunction` set as `queryCar`; `invokerIdentity` set as `admin@org1.example.com`; and `chaincodeArguments` set as an array that contains the asset to query in this run. 
+
+The function should look like this:
+```bash
+    async submitTransaction() {
+        const randomId = Math.floor(Math.random()*this.roundArguments.assets);
+        const myArgs = {
+            contractID: this.roundArguments.contractID,
+            contractFunction: 'queryCar',
+            invokerIdentity: 'Admin@org1.example.com',
+            contractArguments: [`${clientIdx}_${randomId}`],
+            readOnly: true
+        };
+
+        await this.sutAdapter.sendRequests(myArgs);
+    } 
+```
+
+
+### cleanupWorkloadModule
+
+This function is used to clean up after a test as it deletes the assets created in the `initializeWorkloadModule` function. 
+
+We will not be using this function this time as the smart contract we are testing doesn't have a function to delete the assets. So this function will remain empty. 
+
+However, if you are using a different smart contract to test that does have a delete function you can use the same for loop from the `initializeWorkloadModule` phase. You only need to modify the `contractFunction` to call the name of the delete function from the smart contract. 
+
+### Resulting File:
+
+The test callback file should now be fully populated. 
+
+The whole file should look like this:
+
+```bash
+'use strict';
+
+const { WorkloadModuleInterface } = require('@hyperledger/caliper-core');
+
+class MyWorkload extends WorkloadModuleInterface {
+    constructor() {
+        super();
+    }
+    
+    async initializeWorkloadModule(workerIndex, totalWorkers, roundIndex, roundArguments, sutAdapter, sutContext) {
+        await super.initializeWorkloadModule(workerIndex, totalWorkers, roundIndex, roundArguments, sutAdapter, sutContext);
+        this.contractID = 'fabcar';
+        this.version = '0.0.1';
+
+        const requests = [];
+        for (let i=0; i<this.roundArguments.assets; i++) {
+            const assetID = `${this.workerIndex}_${i}`;
+            console.log(`Client ${this.workerIndex}: Creating asset ${assetID}`);
+            const args = {
+                contractID: this.roundArguments.contractID,
```

Comment:
property name expected by the connector is `contractId`. May we should also change the roundArguments version as well ?

Reply:
yeah, the symmetry there is good 👍